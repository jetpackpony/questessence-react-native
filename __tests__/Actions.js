import {
  startQuest,
  answerQuestion,
  showCorrectAnswer,
  goToNextQuestion
} from '../actions/Actions.js';

it('create a START_QUEST action', () => {
  expect(startQuest(0)).toMatchSnapshot();
});

it('create a ANSWER_QUESTION action', () => {
  expect(answerQuestion(0, 0, '8')).toMatchSnapshot();
});

it('create a SHOW_CORRECT_ANSWER action', () => {
  expect(showCorrectAnswer(0)).toMatchSnapshot();
});

it('create a GOTO_NEXT_QUESTION action', () => {
  expect(goToNextQuestion(0)).toMatchSnapshot();
});
