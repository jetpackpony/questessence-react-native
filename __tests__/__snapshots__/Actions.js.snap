// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create a ANSWER_QUESTION action 1`] = `
Object {
  "answer": "8",
  "questId": 0,
  "questionId": 0,
  "type": "ANSWER_QUESTION",
}
`;

exports[`create a GOTO_NEXT_QUESTION action 1`] = `
Object {
  "questId": 0,
  "type": "GOTO_NEXT_QUESTION",
}
`;

exports[`create a SHOW_CORRECT_ANSWER action 1`] = `
Object {
  "questId": 0,
  "type": "SHOW_CORRECT_ANSWER",
}
`;

exports[`create a START_QUEST action 1`] = `
Object {
  "questId": 0,
  "type": "START_QUEST",
}
`;
